tmp/ordinal_classification_demo.ipynb

```diff
@@ -2,16 +2,16 @@
  "cells": [
```

Comment:
I think we'd want to remove this file?

Reply:
I kept it so that you could see the results on monotonicity. I am going to remove it as soon as we confirm the rest